nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-controller-service/src/main/java/org/apache/nifi/controller/kudu/KuduLookupService.java

```diff
@@ -317,7 +317,7 @@ private RecordSchema kuduSchemaToNiFiSchema(Schema kuduTableSchema, List<String>
                 case BINARY:
                 case STRING:
                 case DECIMAL:
-                    fields.add(new RecordField(cs.getName(), RecordFieldType.STRING.getDataType()));
+                    fields.add(new RecordField(cs.getName(), RecordFieldType.DECIMAL.getDataType()));
```

Comment:
I don't know anything about Kudu, so I may put this ticket on hold and ask for input from nifi-dev on a few of these other components (ex Hive)